database/migrations/2021_07_21_151624_create_products_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use App\Enums\StatusType;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateProductsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->text('description');
+            $table->decimal('price',10,2);
```

Comment:
Also, with respect to what I shared about saving money related values in decimal point, please rectify this to integer and add the necessary setter and getter methods in the product model class to take care of the computation as the price is being saved and retrieved respectfully.